amy/templates/reports/membership_trainings_stats.html

```diff
@@ -35,6 +36,9 @@
         <td>{{ result.instructor_training_seats_total }}</td>
         <td>{{ result.instructor_training_seats_utilized }}</td>
         <td>{{ result.instructor_training_seats_remaining }}</td>
+        <td>{{ result._cldt_training_seats_total }}</td>
+        <td>{{ result._cldt_training_seats_utilized }}</td>
+        <td>{{ result._cldt_training_seats_remaining }}</td>
```

Comment:
Django template is complaining about these names. Apparently they can't start with `_`, so my proposition is to use these names instead:

1. `cldt_seats_total_annotation`
2. `cldt_seats_utilized_annotation`
3. `cldt_seats_remaining_annotation`

Changes should be applied here and in `MembershipManager.annotate_with_seat_usage`.

Reply:
I've changed the CLDT annotation variable names but not the others. We can do this after merge/rebase?

Reply:
I think only CLDT annotations started with `_`, so only they needed to be updated with that prefix. Other annotations work just fine because they don't have name clash with model properties.